qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_module.py

```diff
@@ -10,7 +10,7 @@
 from .constants import ModuleKind, SlotNr, MeasurementStatus, ChannelName, ChNr
 
 if TYPE_CHECKING:
-    from .KeysightB1500_base import KeysightB1500
+    import qcodes.instrument_drivers.Keysight.keysightb1500
```

Comment:
so is there no way for sphinx to understand type importd with relative imports? :(

Reply:
Yes that does seem to be a bug/limitation in sphinx